R/class-plotdata.R

```diff
@@ -166,11 +174,6 @@ newPlotdata <- function(.dt = data.table(),
     .dt[[variable.name]] <- gsub(paste0(inferredVarMetadata@variableSpec@entityId, '.'), '', .dt[[variable.name]])
```

Comment:
i don't remember the specifics, but i do remember having a thought about how `melt`ing did or did not reorder the collection vars when they got melted. Anyways, worth eyeballing that the order that comes out of the melt is what you'd like!